src/smt/solver_engine.cpp

```diff
@@ -634,15 +634,6 @@ void SolverEngine::defineFunctionRec(Node func,
   defineFunctionsRec(funcs, formals_multi, formulas, global);
 }
 
-Result SolverEngine::quickCheck()
```

Comment:
Why is this not needed anymore?

Reply:
It actually did nothing but return a (deprecated) entailment result.  This should have been deleted a long time ago.